### PR TITLE
fixed mobile responsiveness, also removed unnecessary code

### PR DESCRIPTION
**Pull request by liamsegura**
# Description

I have cleaned up and replaced a few unnecessary css selectors, as well as fixing the nav bar font and padding.

The Home Screen is now working responsively. Please let me know what you think, and if you'd like me to change anything.

## Screenshots

Screenshot 2023-06-16 at 19 09 05
Screenshot 2023-06-16 at 19 09 16

## Checklist

- [ x ] Tests have been added or updated to cover the changes
- [ x ] Documentation has been updated to reflect the changes
- [ x ] Code follows the established coding style guidelines
- [ x ] All tests are passing

### CODE REVIEW

File: src/components/Navbar/Navbar.css [hunk 1]
```diff
@@ -43,4 +41,28 @@
     height: 4.16vw;
     width: 4.16vw;
     display: block;
-  }
```
Comment (Saksham2401) [inline src/components/Navbar/Navbar.css:46]: great work

**Comment by thestarsahil** [comment]
Can you please update latest the Master branch and then push it again without conflict

**Comment by liamsegura** [comment]
> Can you please update latest the Master branch and then push it again without conflict and possible can you review PR 193

Will have a look once I’m home, once it’s updated and merged I will review #193

**Comment by liamsegura** [comment]
> Can you please update latest the Master branch and then push it again without conflict and possible can you review PR 193

PR should be synced with master

**Comment by thestarsahil** [comment]
Can you able to remove the pages,   actually we want when the user entry into it 
Only login page and signup apper as Like Instagram showing only login and signup page

**Comment by liamsegura** [comment]
> Can you able to remove the pages, actually we want when the user entry into it Only login page and signup apper as Like Instagram showing only login and signup page

Please provide an image as reference

**Comment by liamsegura** [comment]
> Can you able to remove the pages, actually we want when the user entry into it Only login page and signup apper as Like Instagram showing only login and signup page

I have created conditional login for the Navbar component.

"Home" and "About" are not rendered on the home page "/"

Please merge.

**Pull request closed without merging**